# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * crm_enterprise
#
# Translators:
# Jan Horzinka <jan.horzinka@centrum.cz>, 2022
# karol√≠na schusterov√° <karolina.schusterova@vdp.sk>, 2022
# Martin Trigaux, 2022
# Ji≈ô√≠ Podhoreck√Ω, 2022
# Rastislav Brencic <rastislav.brencic@azet.sk>, 2022
# "Tiffany Chang (tic)" <tic@odoo.com>, 2025.
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 16.0+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-10-08 18:46+0000\n"
"PO-Revision-Date: 2025-09-15 11:58+0000\n"
"Last-Translator: \"Tiffany Chang (tic)\" <tic@odoo.com>\n"
"Language-Team: Czech <https://translate.odoo.com/projects/odoo-19/"
"crm_enterprise/cs/>\n"
"Language: cs\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: nplurals=4; plural=(n == 1 && n % 1 == 0) ? 0 : (n >= 2 && n "
"<= 4 && n % 1 == 0) ? 1: (n % 1 != 0 ) ? 2 : 3;\n"
"X-Generator: Weblate 5.12.2\n"

#. module: crm_enterprise
#. odoo-python
#: code:addons/crm_enterprise/tools/business_card_scanner.py:0
msgid "%(contact_name)s's opportunity"
msgstr ""

#. module: crm_enterprise
#. odoo-python
#: code:addons/crm_enterprise/tools/business_card_scanner.py:0
msgid "Could not extract lead information from this image."
msgstr ""

#. module: crm_enterprise
#: model:ir.model,website_form_label:crm_enterprise.model_crm_lead
msgid "Create an Opportunity"
msgstr ""

#. module: crm_enterprise
#: model_terms:ir.ui.view,arch_db:crm_enterprise.crm_lead_view_map
msgid "Customer"
msgstr "Z√°kazn√≠k"

#. module: crm_enterprise
#: model:ir.model.fields,field_description:crm_enterprise.field_crm_lead__days_to_convert
msgid "Days To Convert"
msgstr "Dny k p≈ôevodu"

#. module: crm_enterprise
#: model:ir.model.fields,field_description:crm_enterprise.field_crm_lead__display_name
msgid "Display Name"
msgstr ""

#. module: crm_enterprise
#: model:ir.model.fields,field_description:crm_enterprise.field_crm_lead__days_exceeding_closing
msgid "Exceeded Closing Days"
msgstr "P≈ôekroƒçeny z√°vƒõreƒçn√© dny"

#. module: crm_enterprise
#: model:ir.model.fields,field_description:crm_enterprise.field_crm_lead__id
msgid "ID"
msgstr ""

#. module: crm_enterprise
#. odoo-javascript
#: code:addons/crm_enterprise/static/src/crm_business_card_scanner.xml:0
#: code:addons/crm_enterprise/static/src/crm_business_card_scanner_cog_menu.xml:0
msgid "Import Business Cards"
msgstr ""

#. module: crm_enterprise
#: model:ir.model,name:crm_enterprise.model_crm_lead
msgid "Lead"
msgstr ""

#. module: crm_enterprise
#. odoo-python
#: code:addons/crm_enterprise/tools/business_card_scanner.py:0
msgid "Lead generated from this image."
msgstr ""

#. module: crm_enterprise
#: model_terms:ir.ui.view,arch_db:crm_enterprise.crm_lead_view_pivot
msgid "Leads"
msgstr "Prospekty"

#. module: crm_enterprise
#. odoo-python
#: code:addons/crm_enterprise/tools/business_card_scanner.py:0
msgid "Network error while processing the business cards."
msgstr ""

#. module: crm_enterprise
#. odoo-python
#: code:addons/crm_enterprise/tools/business_card_scanner.py:0
msgid "New Lead Scan"
msgstr ""

#. module: crm_enterprise
#: model_terms:ir.actions.act_window,help:crm_enterprise.crm_opportunity_action_dashboard
msgid "No data found!"
msgstr "Nenalezena ≈æ√°dn√° data!"

#. module: crm_enterprise
#. odoo-python
#: code:addons/crm_enterprise/tools/business_card_scanner.py:0
msgid ""
"Oops, the AI service is not working ü§ñ. Please contact your administrator to "
"verify settings."
msgstr ""

#. module: crm_enterprise
#: model_terms:ir.ui.view,arch_db:crm_enterprise.crm_lead_view_cohort
#: model_terms:ir.ui.view,arch_db:crm_enterprise.crm_opportunity_view_graph
msgid "Opportunities"
msgstr "P≈ô√≠le≈æitosti"

#. module: crm_enterprise
#: model:ir.actions.act_window,name:crm_enterprise.crm_opportunity_action_dashboard
msgid "Pipeline Analysis"
msgstr "Anal√Ωza obchodn√≠ n√°stƒõnky"

#. module: crm_enterprise
#. odoo-javascript
#: code:addons/crm_enterprise/static/src/crm_business_card_scanner.xml:0
msgid "Scan Business Cards"
msgstr ""

#. module: crm_enterprise
#. odoo-python
#: code:addons/crm_enterprise/models/crm_lead.py:0
msgid ""
"The AI agent was not able to generate a lead from the provided image. *sad "
"robot noises* ü§ñ"
msgstr ""

#. module: crm_enterprise
#: model_terms:ir.actions.act_window,help:crm_enterprise.crm_opportunity_action_dashboard
msgid "Use this menu to have an overview of your Pipeline."
msgstr "Pomoc√≠ t√©to nab√≠dky z√≠sk√°te p≈ôehled o va≈°√≠ Pipeline."

#. module: crm_enterprise
#: model_terms:ir.ui.view,arch_db:crm_enterprise.crm_lead_view_graph
msgid "leads"
msgstr "potenci√°ln√≠ z√°kazn√≠ci"

#~ msgid "CRM Activity Analysis"
#~ msgstr "CRM anal√Ωza ƒçinnosti"

#~ msgid "Is Won"
#~ msgstr "Vyhr√°l"

#~ msgid "Lead/Opportunity"
#~ msgstr "Potenci√°ln√≠ z√°kazn√≠k / p≈ô√≠le≈æitost"

#~ msgid "Lost"
#~ msgstr "Ztracen√Ω"

#~ msgid "Pending"
#~ msgstr "ƒåekaj√≠c√≠"

#~ msgid "Won"
#~ msgstr "Vyhr√°no"
